x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/table/title_with_explanation.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { ReactElement } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { EuiPopover, EuiIcon, EuiText } from '@elastic/eui';
+import { useBoolean } from '@kbn/react-hooks';
+import { css } from '@emotion/react';
+
+interface Props {
+  name?: string;
+  description: ReactElement | string;
+  formula: string;
+  'data-test-subj'?: string;
+}
+
+function TitleWithPopoverExplanation(props: Props) {
+  const { name, description, formula } = props;
+  const [isPopoverOpen, { off: closePopover, toggle: togglePopover }] = useBoolean(false);
+
+  return (
+    <>
+      {name && <>{name}</>}
+      <EuiPopover
+        panelPaddingSize="s"
+        focusTrapProps={{
+          returnFocus: false,
+        }}
+        button={
+          <button
+            onClick={(e) => {
+              e.stopPropagation();
+              togglePopover();
+            }}
+            css={css`
+              display: flex;
+            `}
+            data-test-subj={props['data-test-subj']}
+          >
+            <EuiIcon type="questionInCircle" />
+          </button>
+        }
+        isOpen={isPopoverOpen}
+        offset={10}
+        closePopover={closePopover}
+        repositionOnScroll
+        anchorPosition="upCenter"
+        panelStyle={{ maxWidth: 350 }}
+      >
+        <EuiText size="xs">
+          <p>{description}</p>
+          <p>
+            <strong>
+              <FormattedMessage
+                id="xpack.apm.multiSignal.tooltip.formula"
+                defaultMessage="Formula Calculation:"
+              />
+            </strong>
+            <br />
+            <code
+              css={css`
+                word-break: break-word;
+              `}
+            >
+              {formula}
+            </code>
+          </p>
+        </EuiText>
+      </EuiPopover>
+    </>
+  );
+}
+
+interface ExplanationProps {
+  includeTitle: boolean;
+}
+
+export function LogRateWithExplanation({ includeTitle }: ExplanationProps) {
+  return (
+    <TitleWithPopoverExplanation
+      name={
+        includeTitle
```

Comment:
The boolean is a bit confusing here. 

It think it would make the component more flexible and more readable if the component has a default name  with the option for the consumer to pass any name.
```
interface ExplanationProps {
  name?: string;
}
```
`<LogRateWithExplanation name|title={"Log Rate"} /> `